tests/unit/dojson/test_dojson_hep_bd01x09x.py

```diff
@@ -593,6 +593,39 @@ def test_report_numbers_from_037__a():
     assert expected == result['037']
 
 
+def test_report_numbers_from_two_037__a():
+    schema = load_schema('hep')
+    subschema = schema['properties']['report_numbers']
+
+    snippet = (
+        '<datafield tag="037" ind1=" " ind2=" ">'
+        '  <subfield code="a">UTPT-89-27</subfield>'
+        '</datafield>'
+        '<datafield tag="037" ind1=" " ind2=" ">'
+        '  <subfield code="a">CALT-68-1585</subfield>'
+        '</datafield>'
+    )  # record/26564
+
+    expected = [
```

Comment:
You shouldn't discard report numbers! `report_numbers` is a list for a good reason: so that we can place several values inside :wink:
You should put both in `expected` and make sure they round-trip.